doc/manual/src/release-notes/rl-next.md

```diff
@@ -0,0 +1,8 @@
+# Release X.Y (202?-??-??)
+
+- Fixed a bug where `nix-env --query` ignored `--drv-path` when `--json` was set.
+
+- Introduced the store [`mounted-ssh-ng://`](@docroot@/command-ref/new-cli/nix3-help-stores.md).
+  This store allows full access to a Nix store on a remote machine and additionally requires that the store be mounted in the local filesystem.
+
+- [`nix-shell`](../command-ref/nix-shell.md) shebangs now interpret "-E" with relative paths
```

Comment:
```suggestion
- [`nix-shell`](../command-ref/nix-shell.md) shebangs now interpret relative paths in expressions passed to `-E` with the script file's location as the base directory rather than the caller's current working directory.
```

Reply:
> Seems weird that there are two places for release notes now.

Just follow https://github.com/NixOS/nix/blob/master/doc/manual/src/contributing/hacking.md#add-a-release-note
`rl-next.md` should not be committed and has been added to `.gitignore`.
Please remove `rl-next.md` from the commits.


For context, we changed the way we write release notes today, but I believe the above is all you need to know.
 - #9393

Reply:
@matthewbauer in that case, pick any of my suggestions for the one true release note.

Reply:
I was trying to update it to the new style. But then Github wasn't registering my pushes to the branch. I'll fixup and merge later tonight if still needed.

Reply:
fixed the git problems, rl-next issues and incorporated @fricklerhandwerk's suggestions